modules/persistent-storage-csi-dynamic-provisioning-aws-efs.adoc

```diff
@@ -5,7 +5,7 @@
 
 :_content-type: PROCEDURE
 [id="csi-dynamic-provisioning-aws-efs_{context}"]
-= Dynamic provisioning for AWS EFS
+= Dynamic provisioning for Amazon EFS
 
 [role="_abstract"]
 The AWS EFS CSI Driver supports a different form of dynamic provisioning than other CSI drivers. It provisions new PVs as subdirectories of a pre-existing EFS volume. The PVs are independent of each other. However, they all share the same EFS volume. When the volume is deleted, all PVs provisioned out of it are deleted too.
```

Comment:
Doesn't this need to address "The Amazon EFS CSI Driver supports...."?

Reply:
I believe so.

Reply:
Updated Line 8- > Dynamic provisioning for Amazon Elastic File Storage
Updated Line 11 -> "The [AWS EFS CSI driver](https://github.com/openshift/aws-efs-csi-driver) supports a different form of dynamic provisioning...................."